lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4509,8 +4751,6 @@ HSACopy::syncCopyExt(hc::hcCommandKind copyDir, const hc::AmPointerInfo &srcPtrI
             DBOUT(DB_COPY, "HSACopy::syncCopy(), hsa_amd_memory_async_copy() returns: 0x" << std::hex << hsa_status << std::dec <<"\n");
             throw Kalmar::runtime_exception("hsa_amd_memory_async_copy error", hsa_status);
         }
-        Kalmar::ctx.releaseSignal(signal, signalIndex);
```

Comment:
I believe this is not the right place to deallocate a single - instead this should occur in the HSACopy::dispose() routine.  The new profile mode is implemented inside the dispose routine and uses the timestamps from the signal so requires this change.